### DISCUSSION TITLE
Supersonic Modeling

### DISCUSSION THREAD

**Discussion by hamids23** [GuidodiPasquo/Actively-Stabilized-Model-Rocket-Simulator#7] [General]
Why software doesn't model SUPERSONIC CALCULATIONS ?

**Comment by GuidodiPasquo** [comment]
Because they are a completely different set of aerodynamic calculations and they weren't in the scope of the simulator at the time.

**Reply by hamids23** [reply]
thanks for response and don't you have any plan to add supersonic calculations to it ? it's so useful for high power rocketry; it will be so amazing if you do it because i think it will solve many high power rocketry problems

**Reply by GuidodiPasquo** [reply]
Very unlikely, I have new projects that take most of my time nowadays.
